R/tags.R

```diff
@@ -54,6 +54,19 @@ registerMethods <- function(methods) {
     c("knitr", "knit_print", "shiny.tag"),
     c("knitr", "knit_print", "shiny.tag.list")
   ))
+
+  # TODO: After rlang > 0.4.11 hits CRAN, remove this and replace
+  # with ` #' @importFrom rlang obj_address`
+  # (lionel says rlang:::sexp_address() will be available for the next few years)
+  assign_obj_address()
+  setHook(
+    packageEvent("rlang", "onLoad"),
+    function(...) assign_obj_address()
+  )
```

Comment:
This isn't be necessary, and would actually throw an error if it were to execute. It's not necessary because we can assume that rlang is available when `.onLoad` is called, since it's listed in Imports. As for the error: since the namespace is locked after the `.onLoad` callback executes, the hook's `assign()` would fail after that point.